script/bnb/verify_ynbnbk.s.sol

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.24;
+
+import {IVault} from "lib/yieldnest-vault/src/BaseVault.sol";
+
+import {BaseVerifyScript} from "script/BaseVerifyScript.sol";
+import {IStakerGateway} from "src/interface/external/kernel/IStakerGateway.sol";
+
+// FOUNDRY_PROFILE=mainnet forge script VerifyYnBNBkStrategy
+contract VerifyYnBNBkStrategy is BaseVerifyScript {
+    function symbol() public pure override returns (string memory) {
+        return "ynBNBk";
+    }
+
+    function run() public {
+        _setup();
+        _loadDeployment();
+
+        verify();
+    }
+
+    function verify() internal view {
+        assertNotEq(address(vault), address(0), "vault is not set");
```

Comment:
I can move some of these checks into functions in BaseVerifyScript and reuse it @danoctavian